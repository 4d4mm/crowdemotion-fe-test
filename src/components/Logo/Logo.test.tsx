import React from "react";
import { mount } from "enzyme";
import Logo from "./Logo";

describe("Logo", () => {
  it("renders Logo", () => {
    const Logo = mount(<Logo />);
    //expect(logo).toMatchSnapshot();
  });
});
