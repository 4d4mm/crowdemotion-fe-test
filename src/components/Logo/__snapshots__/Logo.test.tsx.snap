// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders Logo 1`] = `
<Logo>
  <svg
    height="24px"
    version="1.1"
    viewBox="0 0 76 24"
    width="76px"
  >
    <title>
      2EBD3EEC-0BC0-47C7-A29D-82F916856692
    </title>
    <desc>
      Created with sketchtool.
    </desc>
    <defs>
      <linearGradient
        id="linearGradient-1"
        x1="50%"
        x2="50%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#F9BE00"
        />
        <stop
          offset="100%"
          stopColor="#F60000"
        />
      </linearGradient>
      <linearGradient
        id="linearGradient-2"
        x1="50%"
        x2="50%"
        y1="0%"
        y2="99.1422086%"
      >
        <stop
          offset="0%"
          stopColor="#54E42B"
        />
        <stop
          offset="100%"
          stopColor="#10A8B7"
        />
      </linearGradient>
      <linearGradient
        id="linearGradient-3"
        x1="50%"
        x2="50%"
        y1="0%"
        y2="99.5117188%"
      >
        <stop
          offset="0%"
          stopColor="#FF2198"
        />
        <stop
          offset="100%"
          stopColor="#8F21A1"
        />
      </linearGradient>
    </defs>
    <g
      fill="none"
      fillRule="evenodd"
      id="MVP-AUDIT-DESKTOP-all-flow"
      stroke="none"
      strokeWidth="1"
    >
      <g
        fillRule="nonzero"
        id="AUDIT-03-Homepage-button-expanded"
        transform="translate(-57.000000, -43.000000)"
      >
        <g
          id="Vertical-Menu/expanded/organisations-v4"
        >
          <g
            id="EH-isotype"
            transform="translate(57.000000, 43.000000)"
          >
            <g
              id="isotype"
            >
              <g>
                <path
                  d="M50.4338457,11.8208499 L50.4341475,11.844144 C58.1434024,17.488224 65.8555317,17.488224 73.5705355,11.844144 C74.1741684,13.672152 71.9493499,18.178992 69.370975,20.2176 C65.6338515,23.1938088 60.4068763,23.6156944 56.2204003,21.279024 C52.7718296,19.333509 49.9783429,14.8113862 50.4338457,11.8208499 Z M62.5671695,2.13162821e-14 C63.5123844,0.115950467 64.4486486,0.293246522 65.3697509,0.530712 C69.7542068,1.73965544 72.9796753,5.38424354 73.5705355,9.797112 C73.6348465,10.0127018 73.5183518,10.2403057 73.3032123,10.3194 C70.4935014,12.0390771 67.3209321,13.1134403 64.0245118,13.461552 C61.0903926,13.7866843 58.1192115,13.4227884 55.3580676,12.400128 C53.8403619,11.818872 52.4002662,11.060712 50.9170539,10.395216 C50.5634974,10.243584 50.4772642,10.041408 50.5290041,9.64548 C51.3342158,4.81626829 55.1963077,1.023128 60.1353911,0.2106 L61.5237468,0.016848 L62.5671695,2.13162821e-14 Z"
                  fill="url(#linearGradient-1)"
                  id="Combined-Shape"
                />
                <path
                  d="M22.3602885,15.795 C20.5599577,20.0301018 16.3271339,22.7903105 11.632869,22.7903105 C6.938604,22.7903105 2.70578024,20.0301018 0.9054494,15.795 C7.47458976,11.8073847 15.7911481,11.8073847 22.3602885,15.795 Z M12.0726587,-4.97379915e-14 C12.9349914,0.134784 13.7542075,0.219024 14.5734237,0.429624 C20.647689,1.95896093 24.3714156,7.93457149 22.9725448,13.908024 C22.9725448,14.034384 22.8949348,14.15232 22.8431949,14.329224 C16.3186988,8.95825278 6.77120393,8.99748599 0.293193139,14.421888 C0.137065294,13.3861375 0.0391873489,12.3428037 -5.86197757e-14,11.296584 C0.0179535071,6.10572489 3.67043516,1.60127328 8.83891081,0.395928 C9.58914031,0.202176 10.3652398,0.12636 11.1240926,-4.97379915e-14 L12.0726587,-4.97379915e-14 Z"
                  fill="url(#linearGradient-2)"
                  id="Combined-Shape"
                />
                <path
                  d="M48.3965784,12.122136 C48.023077,18.0958412 42.9548345,22.7537507 36.8283844,22.7537507 C30.7019343,22.7537507 25.6336918,18.0958412 25.2601904,12.122136 C32.9532267,12.8718548 40.7035421,12.8718548 48.3965784,12.122136 Z M36.8283844,0.0997897923 C42.9408551,0.0997897923 47.9976431,4.74685455 48.3707084,10.706904 C40.6939009,9.9782601 32.9628679,9.9782601 25.2860604,10.706904 C25.6591257,4.74685455 30.7159137,0.0997897923 36.8283844,0.0997897923 Z"
                  fill="url(#linearGradient-3)"
                  id="Combined-Shape"
                />
              </g>
            </g>
          </g>
        </g>
      </g>
    </g>
  </svg>
</Logo>
`;
